frontend/integration-tests/views/crud.view.ts

```diff
@@ -77,9 +77,15 @@ const actionOnKind = (action: string, kind: string) => {
 export const editHumanizedKind = (kind: string) => actionOnKind(actions.edit, kind);
 export const deleteHumanizedKind = (kind: string) => actionOnKind(actions.delete, kind);
 
-export const clickKebabAction = (resourceName: string, actionLabel: string) => {
-  return rowForName(resourceName)
-    .$('[data-test-id="kebab-button"]')
+export const clickCreateWithYAML = async () => {
+  await browser.wait(until.elementToBeClickable(createYAMLButton));
+  await createYAMLButton.click();
+};
+
+export const clickKebabAction = async (resourceName: string, actionLabel: string) => {
+  const kbBtn = await rowForName(resourceName).$('[data-test-id="kebab-button"]');
+  await browser.wait(until.elementToBeClickable(kbBtn));
+  return await kbBtn
```

Comment:
I don't think you need a `return` here.

Reply:
Yep, verified with a couple of suites in both browsers. Pushing...